app/src/main/res/layout/login_activity.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
actually, we are following naming style `fragment_*` , `activity_*` and `content_*` for layouts